### DISCUSSION TITLE
Can I submit a package if the sandbox test fails?

### DISCUSSION THREAD

**Discussion by marco-eckstein** [microsoft/winget-pkgs#131994] [Q&A]
I am working on creating the manifest for a new package. It has a dependency on `Microsoft.VCRedist.2005.x86` (_Microsoft Visual C++ 2005 Redistributable_). Unfortunately, the installation of this package fails in the Windows Sandbox. It does not matter if I try the x86 or the x64 variant.

If I install with WinGet, the command actually succeeds, but the software is not installed. It does not show in _Apps and Features_, and it will not be listed by `winget list`. (Not returning an error looks like a WinGet bug - should I file an issue?)

If I install with a downloaded executable directly, I get a popup with error 1935. 

On my main Windows system (the one that runs the sandbox), the package is installed.

Error 1935 seems to be common, but I have not found much information in context of the Windows Sandbox. Someone said in a comment about [a similar issue](https://stackoverflow.com/questions/74144261/) that this seems like a bug in Windows Sandbox.

Can I still submit the manifest with a dependency on `Microsoft.VCRedist.2005.x86`?

**Comment by Trenly** [comment]
With the release of the 1.6 client, dependencies are supported. However, error 1935 is specific that one or more requirements for running the app were not met. It would be best to figure out why you are getting this error

You can always submit the manifest and see if it passes the validation pipelines, but be aware that it will also be checked by a moderator and some moderators do use sandbox for testing.

**Reply by marco-eckstein** [reply]
I have no idea why I am getting this error. The pull request: https://github.com/microsoft/winget-pkgs/pull/132553
